volatility/plugins/mftparser.py

```diff
@@ -327,13 +353,22 @@ def process_attr_list(self, bufferas, mft_entry, attributes = [], check = True):
                 thetype = ATTRIBUTE_TYPE_ID.get(int(item.Type), None)
                 if thetype == None:
                     return
-                elif item.Length > 0x20 and thetype in ["STANDARD_INFORMATION", "FILE_NAME"]:
-                    theitem = obj.Object(thetype, vm = bufferas, offset = item.AttributeID.obj_offset)
-                    if thetype == "STANDARD_INFORMATION" and (not check or theitem.is_valid()):
-                        attributes.append(("STANDARD_INFORMATION (AL)", theitem))
-                    elif thetype == "FILE_NAME" and (not check or theitem.is_valid()):
-                        mft_entry.add_path(theitem)
-                        attributes.append(("FILE_NAME (AL)", theitem))
+                # -- The below commented code seems to be a misunderstanding of how ATTRIBUTE_LISTs work, they 
```

Comment:
There was a reason this was done... I think it was due to corrupt records.  Either way, you can remove all comments and commented code.